Modules/ITS/src/TrendingTaskITSTracks.cxx

```diff
@@ -140,17 +140,70 @@ void TrendingTaskITSTracks::storePlots(repository::DatabaseInterface& qcdb)
   int ilay = 0;
   for (const auto& plot : mConfig.plots) {
 
-    int add = 0;
-    double ymin = -10.;
-    double ymax = +10.;
+    int class1 = 0;
+    double ymin = 0.;
+    double ymax = 1.;
     if (plot.name.find("mean") != std::string::npos) {
-      add = 0;
-      ymin = 0.;
-      ymax = 15.;
+      if (plot.name.find("NCluster") != std::string::npos) {
+        class1 = 0;
+        ymin = 0.;
+        ymax = 20.;
+      } else if (plot.name.find("EtaDistribution") != std::string::npos) {
+        class1 = 0;
+        ymin = -1.5;
+        ymax = 1.5;
+      } else if (plot.name.find("PhiDistribution") != std::string::npos) {
+        class1 = 0;
+        ymin = 0.;
+        ymax = 2 * TMath::TwoPi();
+      } else if (plot.name.find("VertexZ") != std::string::npos) {
+        class1 = 0;
+        ymin = -15.;
+        ymax = 15.;
+      } else if (plot.name.find("NVertexContributors") != std::string::npos) {
+        class1 = 0;
+        ymin = 0.;
+        ymax = 100.;
+      } else if (plot.name.find("AssociatedClusterFraction") != std::string::npos) {
+        class1 = 0;
+        ymin = 0.;
+        ymax = 1.;
+      } else if (plot.name.find("Ntracks") != std::string::npos) {
+        class1 = 0;
+        ymin = 0.;
+        ymax = 100.;
+      } else if (plot.name.find("VertexX") != std::string::npos) {
+        class1 = 0;
+        ymin = -15.;
+        ymax = 15.;
+      } else if (plot.name.find("VertexY") != std::string::npos) {
+        class1 = 2;
+        ymin = -15.;
+        ymax = 15.;
+      }
+
     } else if (plot.name.find("stddev") != std::string::npos) {
```

Comment:
now you have `VertexXY` so you need to merge the two. Please put also `ymin = -1`, `ymax=1`.

Reply:
mConfig.plots (not mConfig.datasources) is used here.
so, would merging be appropriate?